packages/@uppy/core/tsconfig.build.json

```diff
@@ -5,6 +5,11 @@
     "rootDir": "./src",
     "resolveJsonModule": false,
     "noImplicitAny": false,
+    "paths": {
```

Comment:
I don't understand why we need this when we have `include` already? The default resolve, aka the package, should never change?

Reply:
That's what the PR description is trying to explain: the default resolve targets the build output, we'd rather target the source.

Reply:
And why only these two packages? What about utils? isn't it a bit error prone to maintain an export list _and_ `paths`? Are there alternatives?

Reply:
> why only these two packages?

Those are the only does of core



> isn't it a bit error prone to maintain an export list _and_ `paths`?

We can discuss this once we get to it (where we’d have a different exports list than what’s inside `lib/`), for now this works just fine and will be more often more correct than the default behavior.

Reply:
> Those are the only deps of core

utils is also a dep of core right? Why not utils then?